<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="txtmostrarDatos">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>9</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="entBuscar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbBuscarPor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Humano Por ID</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Humanos Vivos,Salvados,Eliminados</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Familia Por ID</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amigos por ID</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Humanos por Deporte</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnBuscar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>520</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMostrarArbol">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>260</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mostrar Arbol</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtArbol">
    <property name="geometry">
     <rect>
      <x>415</x>
      <y>10</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbEliminar">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>440</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Corvus Glaive</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Midnight</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nebula</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ebony Maw</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Black Dwarf</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Thanos</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="entEliminarFecha">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>480</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminar">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>500</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbSalvar">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>480</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Ant Man</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ironman</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Thor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Spiderman</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnSalvar">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>520</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entSalvar">
    <property name="geometry">
     <rect>
      <x>522</x>
      <y>520</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="entCantHumanos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>10000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCrearHumanos">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Crear humanos</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entElimNivel">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>520</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbEnviarCorreo">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>350</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Enviar Eliminar Humanos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Enviar Salvar Humanos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Enviar Todo la Humanidad</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnEnviarCorreo">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enviar Correo</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCrearhumanos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>292</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Crear Humanos</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBuscarHumanos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>432</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Buscar Humanos</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblElimHumanos">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>392</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Eliminar Humanos</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSalvarHumanos">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>430</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Salvar Humanos</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEnviarCorreo">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>312</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Enviar correo</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPecVir">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pecados y virtudes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPecar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pecar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBuenasAcciones">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Buenas acciones</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
